build.gradle.kts

```diff
@@ -67,6 +68,7 @@ dependencies {
     // Logging
     implementation("ch.qos.logback:logback-classic:${Versions.logback}")
     implementation("net.logstash.logback:logstash-logback-encoder:${Versions.logstashEncoder}")
+    implementation("org.codehaus.janino:janino:${Versions.janino}")
```

Comment:
Hvor brukes denne?

Reply:
> Hvor brukes denne?

Den trengs for at uttrykket inni evaluator-expression skal kunne evalueres.

Men skal sjekke om vi likegodt kan gjøre dette i StatusPages før jeg evnt merger.